VideoLocker/res/values/strings.xml

```diff
@@ -680,8 +680,8 @@
 
 
     <!-- Discussion/Forum -->
-    <string name="forum_query_no_result">Query <i> \"%1$s\" </i> returns no result.</string>
-    <string name="forum_empty">No result found.</string>
+    <string name="forum_query_no_result">No results found for <i>\"%1$s\"</i>. Try another search term or select a discussion category to browse.</string>
```

Comment:
While we're here, we should fix up these items. They should get comments and switch to the named argument style. 

Also, I'm not sure we need quotes and italics.

So it would be, 
```
No results found for \"{search_query}\"
```